UsageExamples/DateHelperExample.cs

```diff
@@ -0,0 +1,290 @@
+﻿using Alpaca.Markets;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace UsageExamples;
+
+public enum MarketStatus
+{
+    PreMarket,
+    Open,
+    Closed,
+    PostMarket
+}
+
+internal sealed class DateHelperExample
+{
+    public async Task Run()
+    {
+        // old way of finding market status which is time consuming and takes up an api call
+        // my tests took an average of 4000 milliseconds for this task
+        var task1watch = new Stopwatch();
+        task1watch.Start();
+        var oldStatus = await GetAlpacaMarketStatus(DateTime.UtcNow).ConfigureAwait(false);
+        task1watch.Stop();
+        Console.WriteLine($"Old method took {task1watch.ElapsedMilliseconds} milliseconds to run.");
+
+        // new way of finding market status which takes far less time and no api call necessary
+        // my tests took an average of 2 milliseconds for this task
+        var task2watch = new Stopwatch();
+        task2watch.Start();
+        var newStatus = GetMarketStatus(DateTime.UtcNow);
+        task2watch.Stop();
+        Console.WriteLine($"New method took {task2watch.ElapsedMilliseconds} milliseconds to run.");
+    }
+
+    /// <summary>
+    /// Old version taking in a date in local UTC time and returning cuurrent market status
+    /// </summary>
+    /// <param name="date"></param>
+    /// <returns></returns>
+    public static async Task<MarketStatus> GetAlpacaMarketStatus(DateTime date)
+    {
+        var key = new SecretKey("...", "...");
+
+        using var tradingClient = Environments.Paper.GetAlpacaTradingClient(key);
+
+        var calendars = await tradingClient.ListIntervalCalendarAsync(
+            CalendarRequest.GetForSingleDay(new DateOnly(date.Year, date.Month, date.Day))).ConfigureAwait(false);
+
+        var calendar = calendars.Single();
+
+        var easternTime = ConvertToEasternTime(date);
```

Comment:
Why do you need this conversion at all? Just remove it and the related method - it will unblock your PR because I'm unable to merge it while it breaks the build.